src/main/java/org/blockjam/core/BlockJamCorePlugin.java

```diff
@@ -73,6 +77,10 @@ public static ConfigManager config() {
         return instance().configManager;
     }
 
+    public static BungeeManager bungeeManager() {
+        return instance().bungeeManager;
```

Comment:
Make `BungeeManager` a singleton class and give it an `instance()` method as well. Code for accessing the singleton instance would then only need to look like `BungeeManager.instance()`.